MAUI/Popup/styles.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: post
+title: Styles in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Styles support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+--- 
+
+# Styles in .NET MAUI Popup (SfPopup)
+
+The SfPopup applies style to all of its elements by using the `SfPopup.PopupStyle` property.
+
+## Styling popup header
+
+The SfPopup allows customizing the header elements with various header customizations as follows:
```

Comment:
The SfPopup allows customizing the header appearance using the below properties.